contrib/rgbenv.bash

```diff
@@ -0,0 +1,377 @@
+#!/usr/bin/env bash
+
+# SPDX-License-Identifier: MIT
+#
+# Copyright (c) 2022 Zumi Daxuya
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+set -e
+shopt -u nullglob
+
+# --------------- Variables -------------------------------------------------
+
+RGBDS_GIT="https://github.com/gbdev/rgbds"
+
+# The version number is directly appended after these variables
+RGBDS_DOWNLOAD_BASE="https://codeload.github.com/gbdev/rgbds/tar.gz/refs/tags/v"
```

Comment:
> While it's possible to use the codeload URL directly, you generally want to use the API URL, since it handles things like authentication more gracefully, and the codeload URLs for private repositories are generally ephemeral.

<https://stackoverflow.com/questions/60188254/how-is-codeload-github-com-different-to-api-github-com>

Reply:
I'll give that a try

Reply:
Thank you!